tileview/src/main/java/com/qozix/tileview/tiles/Tile.java

```diff
@@ -255,7 +258,12 @@ void reset() {
     mState = State.UNASSIGNED;
     mRenderTimeStamp = null;
     if( mBitmap != null && !mBitmap.isRecycled() ) {
-      mBitmap.recycle();
+      BitmapRecycler recycler = mBitmapRecyclerReference.get();
+      if( recycler != null ) {
+        recycler.recycleBitmap( mBitmap );
+      } else {
```

Comment:
i'd drop the `else` block - they already get a default recycler, if they've nulled it out or whatever then don't do anything...
```
if( recycler != null ) {
  recycler.recycleBitmap( mBitmap );
} // done
```